cms/static/sass/_variables.scss

```diff
@@ -138,6 +141,9 @@ $green-u1: desaturate($green,15%);
 $green-u2: desaturate($green,30%);
 $green-u3: desaturate($green,45%);
 
+$uxpl-green-base: #008a00; // UXPL, also wcag2a compliant
+$uxpl-green-hover-active: lighten($uxpl-green-base, 7%); // UXPL, also wcag2a compliant
+
```

Comment:
I'd prefer to update our `$blue` and `$green` with these values, but it would change all blues and greens. Thoughts?